File: datasets/java/valid/Valid0063.java
public class Valid0063 {
    private int value;
    
    public Valid0063(int value) {
        this.value = value;
    }
    
    public int getValue() {
        return value;
    }
    
    public static void main(String[] args) {
        Valid0063 obj = new Valid0063(42);
        System.out.println("Value: " + obj.getValue());
    }
}

File: datasets/python/valid/valid_0063.py
class Processor:
    def __init__(self, num):
        self._num = num

    def get_num(self):
        return self._num

    def set_num(self, size):
        self._num = size

obj = Processor(44)
print(obj.get_num())
obj.set_num(14)
print(obj.get_num())
